### ISSUE TITLE
Issue with chat dots.

### ISSUE THREAD

**Issue by mayank408** [fossasia/susi_android#299]
**Actual Behaviour**

When the user is not connected to internet and send a message , in reply the susi message keeps on showing the chat dots until the net gets connected.

**Expected Behaviour**
If internet in not connected chat bubbles should not show up.

**Comment by mariobehling** [comment]
As expected behaviour, I would rather expect an information "You are currently not connected". How do chat apps do this?

**Comment by mayank408** [comment]
The snackbar already shows "Internet Connection Not Available " , we just need to stop showing the chat bubbles.

![whatsapp image 2016-10-24 at 9 10 23 pm](https://cloud.githubusercontent.com/assets/22056321/19652433/576696b4-9a2e-11e6-9a02-c967b41104ae.jpeg)

**Comment by mariobehling** [comment]
The top bar shows if there is reception. It can also show reception to Wifi, but this does not mean that there is an Internet connection.

**Comment by mayank408** [comment]
@the-dagger What are your views on this.

**Comment by the-dagger** [comment]
@mayank408 Agreed, the dots shouldn't be displayed if there is no internet connection. :+1:

**Comment by mayank408** [comment]
@the-dagger Pls assign this to me . I would like to work on it.
